src/fundus/publishers/de/postillon.py

```diff
@@ -0,0 +1,41 @@
+import datetime
+from typing import List, Optional
+
+from lxml.cssselect import CSSSelector
+
+from fundus.parser import ArticleBody, BaseParser, ParserProxy, attribute
+from fundus.parser.utility import (
+    extract_article_body_with_selector,
+    generic_author_parsing,
+    generic_date_parsing,
+    generic_topic_parsing,
+)
+
+
+class PostillonParser(ParserProxy):
+    class V1(BaseParser):
+        _paragraph_selector  = CSSSelector("div[id=post-body] > p")
```

Comment:
The Postillon includes blockqutos sometimes, which aren't parsed atm. e.g. https://www.der-postillon.com/2024/04/alltagsfrage-baden-verboten.html
There are two options to fix this as far as I see: `div[id=post-body] p` or `div[id=post-body] > p, div[id=post-body] > blockquote > p`

Reply:
`div[id=post-body] p` seems morally correct here, pushed.